boards/atmega256rfr2-xpro/Makefile.features

```diff
@@ -5,3 +5,4 @@ FEATURES_PROVIDED += periph_i2c
 FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_rtt
```

Comment:
Could this be moved to a more central place, let's say `cpu/atmega_common/Makefile.features` ? (and also keep alphabetical order)

Reply:
Since it only works when the board provides a 32kHz crystal oscillator, I'm afraid not.

Reply:
Ok, since it doesn't work on arduino-uno and arduino-mega2560 let's just provide the features on boards where it was tested. It works on atmega256rfr2-xpro

Reply:
Then just fix the alphabetical order here and in other boards.

Reply:
The alphabetical order has been restored.